CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/DataAccessPluginType.java

```diff
@@ -84,11 +80,9 @@
      */
     private static List<PositionalDataAccessPluginType> getTypesOrderedByPosition() {
         final List<PositionalDataAccessPluginType> ptypeList = new ArrayList<>();
-        Lookup.getDefault().lookupAll(DataAccessPluginType.class).stream().forEach((type) -> {
-            ptypeList.addAll(type.getPluginTypeList());
-        });
+        Lookup.getDefault().lookupAll(DataAccessPluginType.class).stream().forEach(type -> ptypeList.addAll(type.getPluginTypeList()));
```

Comment:
Is the ```stream``` redundant here given ```forEach``` is used right after? I noticed this all throughout this PR - would you mind searching for instances of ```stream().forEach(...)``` and correcting that?

Reply:
forEach is a function of the stream object. Unless there is another forEach function that can be used without the stream object, it will likely need to stay

Reply:
You are correct, and from what I've read it shouldn't make too much difference which one you use for most cases. That being said, unless you're following up with streaming operations, the ```Iterable``` version does seem a little more flexible in that it has order, respects data locks and allows premature returns. @arcturus2 - what do you think?

Reply:
@cygnus-x-1 based on what I've read a stream is less memory intensive that an iterator which writes to the heap more often (for the reasons you have described - the extra checks it does).

I'd like to get into using streams more so I'd favor to leave it as is.

Reply:
Ok I think I'll leave it as is then. We can address this at a later date if we do decide we want to go down a different path

Reply:
@arcturus from what i've read the memory/space improvements only really apply when streaming over large datasets. For a ```Lookup``` as above, where you would expect no more than 10's of results, streams can actually have a higher time and possibly even memory overhead due to the additional objects created by the stream. We should revisit this later...

Reply:
ok @cygnus-x-1 worth fleshing out more sometime.